openid-sharedsignals-framework-1_0.md

```diff
@@ -537,42 +538,34 @@ critical_subject_members
   a Subject Member in an event, MUST be interpreted by a Receiver.
-  
-supported_scopes
 
-> OPTIONAL. A list of OAuth {{RFC6749}} scope names that the Transmitter supports for specific endpoints. The value of this field is a JSON object that has the endpoint names as keys, and arrays of scope name strings they support as their values. OAuth tokens obtained using any of the scopes defined here MUST be accepted by the specified endpoint. Any key that is not defined as an endpoint in the Transmitter Configuration Metadata MUST be ignored. If the supported_scopes member is present in the metadata and an endpoint is not listed as a key, then that endpoint MUST not require OAuth for authorization.
+authorization_schemes
 
-authorization_servers
+> OPTIONAL. A list of complex type that specifies supported
+  authorization scheme properties defined in {{authorization-scheme}}.  To enable seamless discovery of
+  configurations, the service provider SHOULD, with the appropriate
+  security considerations, make the authorization_schemes attribute
+  publicly accessible without prior authentication.
 
-> OPTIONAL. An array of supported authorization servers and the scopes they support. Each element of the array is a Authorization Server Descriptor JSON object defined in the section {{authz-server-descriptor}} below. If the `supported_scopes` member is present in the metadata, then the `authorization_servers` MUST also be present, and it MUST provide a server location for every supported scope.
 
 TODO: consider adding a IANA Registry for metadata, similar to Section 7.1.1 of
 {{RFC8414}}. This would allow other specs to add to the metadata.
 
-### Authorization Server Descriptor {#authz-server-descriptor}
-An Authorization Server Descriptor is a JSON object that has two keys:
+### Authorization scheme {#authorization-scheme}
+SSF is an HTTP based signals sharing framework. It is agnostic to the authentication and authorization schemes used to secure stream configuration APIs. It does not provide any SSF specific authentication and authorization schemes but relies on the cooperating parties mutual security considerations. Authorization scheme section of the metadata, providers discovery information related to the transmitter's stream management APIs.
 
-scopes
+spec_urn  
 
-> REQUIRED. An array of scope names supported by the authorization server
+> REQUIRED. A URN that describes the specification of the protocol being used.
 
-servers
+The receiver will call the transmitter APIs by providing appropriate credentials as per the `spec_urn`.
 
-> REQUIRED. An array of authorization server URLs. This is the URL from which the Authorization Server Metadata MAY be obtained by following the process described in Section 3 of RFC8414 {{RFC8414}}
 
-The following is a non-normative example of an Authorization Server Descriptor
+If the `spec_urn` in Authorization scheme is `urn:ietf:rfc6749`
```

Comment:
Suggested text: "The following is a non-normative example of an Authorization Server Descriptor", but you should probably add an equivalent line for the example you are providing"